v2.0/import.md

```diff
@@ -172,14 +172,17 @@ URLs for the file you want to import and your temp directory must use the follow
 | Google Cloud [<sup>1</sup>](#notes) | `gs` | Bucket name | None |
 | HTTP [<sup>2</sup>](#notes) | `http` | Remote host | N/A |
 | NFS/Local [<sup>3</sup>](#notes) | `nodelocal` | File system location | N/A |
+| S3-compatible services | `s3` | Bucket name | `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `AWS_ENDPOINT` |
 
 #### Notes
```

Comment:
Since this is called `Considerations` in `backup.md`, perhaps use that heading here as well? Also use bullets like in `backup.md`?